src/state_machine/coordinator/fire.rs

```diff
@@ -1276,6 +1336,14 @@ impl<Aggregator: AggregatorTrait> CoordinatorTrait for Coordinator<Aggregator> {
         self.config.clone()
     }
 
+    fn get_config_mut(&mut self) -> &mut Config {
+        &mut self.config
+    }
+
+    fn set_coordinator_public_key(&mut self, key: Option<ecdsa::PublicKey>) {
+        self.coordinator_public_key = key;
+    }
```

Comment:
Why take an option here, instead of requiring a public key? To allow for completely unsetting the coordinator's public key?

Reply:
Yes, it seems strange to allow setting but not unsetting, hence passing an `Option` to allow unsetting.